test/Faker/Container/ContainerTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Container;
+
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+class ContainerTest extends TestCase
```

Comment:
```suggestion
final class ContainerTest extends TestCase
```

Reply:
We should really have a CS rule for this..